osvr/RenderKit/DirectModeVendors.h

```diff
@@ -238,7 +238,8 @@ namespace renderkit {
             Vendor{"VRG", "VRGate"},
             Vendor{"TSB", "VRGate"},
             Vendor{"VRV", "Vrvana"},
-            Vendor{"TVR", "TotalVision"},
+            Vendor{"TVR", "TotalVision", "Some TotalVision LDT HMD's  with early firmware/EDID data"},
+            Vendor{"TVL", "TotalVision", "TotalVision LDT HMD units"},
```

Comment:
I'd suggest not populating that third field - just leaving it `Vendor{"TVL", "TotalVision"}` (which will auto-fill the third field's role as "TotalVision"). That will future-proof things for you.